main.go

```diff
@@ -45,12 +46,27 @@ func usage() {
 	os.Exit(2)
 }
 
+type ExitError struct {
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ExitError should have comment or be unexported